scripts/offline-elf-inspect.sh

```
@@ -190,30 +190,28 @@ fi
         HSACO_NAME=$(basename "${hsaco}")
         MCPU="${HSACO_NAME%.hsaco}"
         log_info "Disassembling ${HSACO_NAME} with --mcpu=${MCPU}..."
 
-        DISASM_SAMPLE=$("${LLVM_OBJDUMP}" -d --mcpu="${MCPU}" "${hsaco}" 2>/dev/null | grep -iE 'v_wmma|v_mfma|s_delay_alu|v_dot4' | head -n 15 || true)
+        DISASM_SAMPLE=$("${LLVM_OBJDUMP}" -d --mcpu="${MCPU}" "${hsaco}" 2>/dev/null | grep -iE 'v_wmma|v_mfma|s_delay_alu|v_dot4' || true)
 
         if [ -n "${DISASM_SAMPLE}" ]; then
             log_success "Found lowered matrix hardware intrinsics in ${HSACO_NAME}:"
-            echo "${DISASM_SAMPLE}"
+            echo "${DISASM_SAMPLE}" | head -n 15
+            INSTRINSICS_FOUND=1
```

Shellcheck (0.11.0)

[warning] INSTRINSICS_FOUND appears unused. Verify use (or export if used externally).

(SC2034)